### ISSUE TITLE
Can't customize current_user method

### ISSUE THREAD

**Issue by mtsmfm** [bm-sms/bbs#41]
I have following configuration:

```
config.current_user = :current_admin_user
```

I encountered following error when I accessed http://localhost:3000/bbs/categories/1/topics:

```
onView::Template::Error (undefined local variable or method `current_user' for #<Bbs::TopicsController:0x00561e0328b8c0>):
 %h1.bbs-topic__title= t('.title', category: @category.name)
    3:
    4:   .bbs-new-topic
    5:     - if logged_in?
    6:       = link_to t('.new_topic'), new_category_topic_path(@category), class: 'bbs-button'
    7:     - else
    8:       = link_to t('.require_login'), Bbs.config.login_path, class: 'bbs-button'

vendor/bbs/app/controllers/bbs/application_controller.rb:9:in `logged_in?'
undle/gems/actionpack-5.0.0.1/lib/abstract_controller/helpers.rb:68:in `logged_in?'
vendor/bbs/app/views/bbs/topics/index.html.haml:5:in `_vendor_bbs_app_views_bbs_topics_index_html_haml___494871308054293833_47343449275000'
```

**Comment by kajisha** [comment]
Thank you for your report.
I'll fix it.

**Comment by kajisha** [comment]
I think this issue fix in 1.0.1.
